### DISCUSSION TITLE
Deploy with Docker + HTTPS

### DISCUSSION THREAD

**Discussion by lcymicwork** [open-webui/open-webui#967] [Q&A]
is there any simple way to enable HTTPS? The current docker come with HTTP only.

**Comment by justinh-rahb** [comment]
Front it with any of the many great reverse proxy servers of your choice. Not in scope to be included in the project, there's Docker one-liners that will set the whole shebang up for you.

**Comment by UberMetroid** [comment]
I use a cloud flare tunnel. pretty easy.

**Reply by UserName-wang** [reply]
Can you please share your solution? thank you!
